--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,6 +11,7 @@
 set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMakeModules")
 set(OpenGL_GL_PREFERENCE "LEGACY")
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+set(CEF_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/build/cef")
 
 # puts the app in demomode, which records a 5 second webm video and quits
 if(NOT DEMOMODE)
@@ -108,7 +109,8 @@
 endif()
 message(STATUS "Using CEF for ${CMAKE_SYSTEM_NAME} - ${CEF_PLATFORM} (${CEF_ARCH_DETECTION})")
 include(DownloadCEF)
-DownloadCEF("${CEF_PLATFORM}" "${CEF_VERSION}" "${CMAKE_CURRENT_BINARY_DIR}/cef")
+# DISABLED as it's downloaded using xbpc-src
+# DownloadCEF("${CEF_PLATFORM}" "${CEF_VERSION}" "${CMAKE_CURRENT_BINARY_DIR}/cef")
 
 # add cef's cmake files to the lookup so we can use those directly
 list(APPEND CMAKE_MODULE_PATH "${CEF_ROOT}/cmake")
 